SUMMARY.md

```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](backend/neo4j-database.md)
   * [GraphQL](backend/graphql.md)
-  * [Legacy Migration](backend/db-migration-worker/README.md)
+  * [Testing](backend/testing.md)
+  * [Middleware](backend/middleware.md)
+  * [Legacy Migration](backend/db-migration-worker.md)
```

Comment:
Same, legacy migration is completely irrelevant for open source contributors. It's being moved into folder `deployment` in https://github.com/Human-Connection/Human-Connection/pull/485